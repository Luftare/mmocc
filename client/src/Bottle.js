import React from 'react';

const Bottle = ({ amountLeft }) => {
  const percentLeft = Math.floor(amountLeft * 100);
  return <span>{percentLeft} %</span>;
};

export default Bottle;
