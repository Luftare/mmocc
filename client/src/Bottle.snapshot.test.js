import React from 'react';
import Bottle from './Bottle';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';

import { configure, shallow } from 'enzyme';

configure({ adapter: new Adapter() });

describe('Bottle snapshot', () => {
  it('renders correnctly', () => {
    const bottle = renderer.create(<Bottle amountLeft={0} />);
    expect(bottle).toMatchSnapshot();
  });
});
