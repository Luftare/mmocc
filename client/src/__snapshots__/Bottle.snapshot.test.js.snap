// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bottle snapshot renders correnctly 1`] = `
<span>
  0
   %
</span>
`;
